app/javascript/packs/navbar.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+
+import styled from 'styled-components'
+import { Container, BrandBar, ControlBar, BrandImage, BrandText, ToggleText, ToggleWrapper, Button} from '../packs/style.jsx'
+
+import DonationJar from '../../assets/images/icn-donation-jar@1x.png'
+import Toggle from 'brave-ui/components/formControls/toggle'
+
+export default class Navbar extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render(){
+    return(
+      <Container>
+
+        <BrandBar mode={this.props.mode}>
+          <BrandImage src={DonationJar}/>
+          <BrandText>Tipping Banner</BrandText>
```

Comment:
Let's ensure that we're internationalizing our javascript components

Reply:
Will make an item for i18n, it extends beyond this feature